config/nginx/conf.d/qa_local.conf

```diff
@@ -29,6 +29,7 @@ server {
     proxy_set_header Host $host;
     proxy_redirect off;
     proxy_cookie_path / "/; HTTPOnly; Secure";
+    client_max_body_size       100m;
```

Comment:
This doesn't seem large enough for projects with multi-media files...?

Reply:
Suggestion?